packages/patternfly-4/react-integration/demo-app-ts/src/components/demos/ChipGroupDemo/ChipGroupDemo.tsx

```diff
@@ -0,0 +1,67 @@
+import { Badge, Chip, ChipGroup } from '@patternfly/react-core';
+import React, { Component } from 'react';
+
+export interface BadgeChipProps extends React.HTMLProps<HTMLDivElement> {
```

Comment:
I expected that the demo would import the props defined and exported in Chip.tsx rather than declare new ones. @dlabaj is there an ideal way to handle this?